readme.md

```diff
@@ -76,6 +76,8 @@ It works on macOS, Linux, and Windows 7+.
 
 **Note**: On Windows, there isn't a clear notion of a "Window ID". Instead it returns the memory address of the window "handle" in the `id` property. That "handle" is unique per window, so it can be used to identify them. [Read more…](https://msdn.microsoft.com/en-us/library/windows/desktop/ms632597(v=vs.85).aspx#window_handle).
 
+**Note** Depending on OS, getting various parts of this information might require additional rights the user doesn't have or be impossible(in general or at the moment of time). In such cases the information is provided on best effor basis, with missing pieces being `undefined`.
```

Comment:
You need to be more specific about what could be missing and for what OSes? For example, for macOS it's not really possible that individual properties do not exist. This information is important for the user, but also for creating proper types (https://github.com/sindresorhus/active-win/pull/40).

Reply:
I wasn't clear, it returns whole result as `undefined`, so that part of readme isn't needed anymore.